<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MiniBasic</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>185</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 30, 30);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnLoad">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border: 0px solid white;
image: url(:/MainWindow/image/load.png);
}
QPushButton:hover {
border: 0px solid white;
image: url(:/MainWindow/image/load_hover.png);
}
QPushButton:pressed {
border: 0px solid white;
image: url(:/MainWindow/image/load_pressed.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRun">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border: 0px solid white;
image: url(:/MainWindow/image/run.png);
}
QPushButton:hover {
border: 0px solid white;
image: url(:/MainWindow/image/run_hover.png);
}
QPushButton:pressed {
border: 0px solid white;
image: url(:/MainWindow/image/run_pressed.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>45</width>
         <height>45</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border: 0px solid white;
image: url(:/MainWindow/image/clear.png);
}
QPushButton:hover {
border: 0px solid white;
image: url(:/MainWindow/image/clear_hover.png);
}
QPushButton:pressed {
border: 0px solid white;
image: url(:/MainWindow/image/clear_pressed.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
                     </property>
                     <property name="text">
                      <string>  代码  </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTextBrowser" name="code">
                     <property name="styleSheet">
                      <string notr="true">QTextBrowser {
background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
border: 3px solid rgb(60, 60, 60);
}
QScrollBar {
background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
border: 0px solid rgb(60, 60, 60);
margin: 16px 0px 16px 0px;
}
QScrollBar::handle {
background-color: rgb(90, 90, 90);
margin: 0px 3px 0px 3px;
}
QScrollBar::handle:hover {
background-color: rgb(120, 120, 120);
}
QScrollBar::sub-line {
subcontrol-position: top;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::add-line {
subcontrol-position: bottom;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::sub-page,
QScrollBar::add-page {
background-color: rgb(60, 60, 60);
}
</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_10">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
                     </property>
                     <property name="text">
                      <string>  语法树  </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTextBrowser" name="tree">
                     <property name="styleSheet">
                      <string notr="true">QTextBrowser {
background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
border: 3px solid rgb(60, 60, 60);
}
QScrollBar {
background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
border: 0px solid rgb(60, 60, 60);
margin: 16px 0px 16px 0px;
}
QScrollBar::handle {
background-color: rgb(90, 90, 90);
margin: 0px 3px 0px 3px;
}
QScrollBar::handle:hover {
background-color: rgb(120, 120, 120);
}
QScrollBar::sub-line {
subcontrol-position: top;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::add-line {
subcontrol-position: bottom;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::sub-page,
QScrollBar::add-page {
background-color: rgb(60, 60, 60);
}
</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>  运行结果  </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextBrowser" name="run">
                  <property name="styleSheet">
                   <string notr="true">QTextBrowser {
background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
border: 3px solid rgb(60, 60, 60);
}
QScrollBar {
background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
border: 0px solid rgb(60, 60, 60);
margin: 16px 0px 16px 0px;
}
QScrollBar::handle {
background-color: rgb(90, 90, 90);
margin: 0px 3px 0px 3px;
}
QScrollBar::handle:hover {
background-color: rgb(120, 120, 120);
}
QScrollBar::sub-line {
subcontrol-position: top;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::add-line {
subcontrol-position: bottom;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::sub-page,
QScrollBar::add-page {
background-color: rgb(60, 60, 60);
}
</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
               </property>
               <property name="text">
                <string>  控制台  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextBrowser" name="console">
               <property name="styleSheet">
                <string notr="true">QTextBrowser {
background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
border: 3px solid rgb(60, 60, 60);
}
QScrollBar {
background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
border: 0px solid rgb(60, 60, 60);
margin: 16px 0px 16px 0px;
}
QScrollBar::handle {
background-color: rgb(90, 90, 90);
margin: 0px 3px 0px 3px;
}
QScrollBar::handle:hover {
background-color: rgb(120, 120, 120);
}
QScrollBar::sub-line {
subcontrol-position: top;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::add-line {
subcontrol-position: bottom;
subcontrol-origin: margin;
height: 16px;
}
QScrollBar::sub-page,
QScrollBar::add-page {
background-color: rgb(60, 60, 60);
}
</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="acceptRichText">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
            </property>
            <property name="text">
             <string>  命令行输入  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cmd">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
font: 15pt &quot;Microsoft YaHei UI&quot;;
border: 3px solid rgb(60, 60, 60);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
